packages/core/contracts/oracle/implementation/VotingV2.sol

```diff
@@ -1093,9 +1053,13 @@ contract VotingV2 is
     function retrieveRewardsOnMigratedVotingContract(
         address voterAddress,
         uint256 roundId,
-        VotingAncillaryInterface.PendingRequestAncillary[] memory toRetrieve
+        MinimumVotingAncillaryInterface.PendingRequestAncillary[] memory toRetrieve
```

Comment:
Really blown away that the size of the interface being imported matters at all 🤦

Reply:
with the "reduced" interface:
```
Contract is 24544 bytes in size.
This leaves a total of 32 bytes within the EIP170 limit.
```

with the "standard" interface:
```
Contract is 24621 bytes in size.
This leaves a total of -45 bytes within the EIP170 limit.
```

Reply:
I think the reason is that the standard interface imports `FixedPoint`.